test/controller/objecting.entity.name.controller.spec.unit.ts

```diff
@@ -188,6 +194,29 @@ describe("objecting entity name tests", () => {
 
     expect(response.status).toEqual(302);
     expect(response.header.location).toEqual(OBJECTIONS_COMPANY_NUMBER);
+    expect(mockRetrieveFromObjectionSession).toHaveBeenCalledTimes(1);
+    expect(mockRetrieveAccessToken).not.toBeCalled();
+    expect(mockRetrieveCompanyProfileFromSession).not.toBeCalled();
+  });
+
+  it("should render the company number page when posting with entered details change answers false", async () => {
```

Comment:
similarly to above, could be named "should render the company number page when posting with entered details and change answers flag set to false"